tutorial/VisualizingQuantumState.playground/Contents.swift

```diff
@@ -0,0 +1,168 @@
+/*:
```

Comment:
This playground is a very simplified version of the [equivalent jupyter notebook](https://github.com/QISKit/qiskit-tutorial/blob/master/1_introduction/visualizing_quantum_state.ipynb), it lacks many of the explanations included in the former. Hope it is enough.